app/controllers/darjeelink/application_controller.rb

```diff
@@ -9,11 +9,20 @@ class ApplicationController < ActionController::Base
 
     private
 
+    def determine_ip_to_verify
```

Comment:
* Robocop updated 'rules' got sad about this. Did not feel the original was hard to read or complex, but fixed them up to make it happy anyway...